### PR TITLE
mutagen: Revert 1.34

### PR DESCRIPTION
**Pull request by the-maldridge**
The latest version of mutagen broke picard.

### CODE REVIEW

**Comment by Vaelatern** [comment]
More information, to document this?

**Comment by the-maldridge** [comment]
Mutagen 1.33->1.34 removes the BitPaddedInt class which is needed for picard to properly function.  To confirm this, simply install picard and try to launch it.

File: srcpkgs/mutagen/template [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Template file for 'mutagen'
 pkgname=mutagen
-version=1.34
+reverts="1.34_1"
+version=1.33
```
Comment (Vaelatern) [inline srcpkgs/mutagen/template]: We were on 1.33.2.

See a25367a490beec3f31e262db451cb7680e7e67df

**Comment by chneukirchen** [comment]
Shouldn't we rather patch Picard?

https://github.com/metabrainz/picard/pull/479/commits/289925e2951b61b06b365d2d031ef0d73f4a33e5

**Comment by the-maldridge** [comment]
@chneukirchen That would be a good solution as far as I'm concerned.

**Comment by the-maldridge** [comment]
meh, at least my local version isn't broken.

**Pull request closed without merging**